L1Trigger/TrackFindingTracklet/src/TrackletLUT.cc

```diff
@@ -478,6 +478,43 @@ void TrackletLUT::initProjectionBend(double k_phider,
   writeTable();
 }
 
+void TrackletLUT::initProjectionDiskRadius(int nrbits) {
+  double roffset = 3.0;
+
+  for (unsigned int ir = 0; ir < (1u << nrbits); ir++) {
+    double r = ir * settings_.rmaxdisk() / (1u << nrbits);
+
+    int rbin1 = 8.0 * (r - roffset - settings_.rmindiskvm()) / (settings_.rmaxdisk() - settings_.rmindiskvm());
+    int rbin2 = 8.0 * (r + roffset - settings_.rmindiskvm()) / (settings_.rmaxdisk() - settings_.rmindiskvm());
+
+    if (rbin1 < 0) {
+      rbin1 = 0;
+    }
+    rbin2 = clamp(rbin2, 0, 7);
```

Comment:
Unnamed hard-wired constants, such as 7, 15 & 64 are not allowed.

Reply:
I removed the few that I could. I'll need @aryd help with some of the detector specific ones.

Reply:
Someone (Brent?) fixed this.